05-angularjs/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html ng-app="appMovie">
+<head>
+	<title>Topic 5 - AngularJS</title>
+
+	<script src="node_modules/angular/angular.min.js"></script>
+	<script src="js/script.js"></script>
+	<script src="js/moviesBD.js"></script>
+
+	<link rel="stylesheet" type="text/css" href="mystyle.css">
+
+</head>
+<body>
+
+	<section ng-controller="movieCtrl">
+
+		<section id="details">
+			<p id="detSub">{{getDetails}} </p> 
+			{{getTittle}} <br>
+			{{getDirector}} <br>
+			{{getDuration}} <br><br>
+			<img src={{getPhoto}}  width="250" height="250" >
```

Comment:
This is not ideal, it is possible that {{getPhoto}} is not defined. Use the ng-src directive to work around this issue.